geo-location.html

```diff
@@ -1,217 +1,174 @@
-<!-- Copyright Eric Bidelman <ebidel@gmail.com> -->
-
-<link rel="import" href="../polymer/polymer.html">
-
-<!-- TODO: decide if we need to cache results across instances. -->
-
-<!--
-Provides the current location using the [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation).
-
-<b>Example - basic</b>
-
-    <geo-location></geo-location>
-
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
+<!--<link rel='import' href='../bower_components/polymer/lib/elements/dom-if.html'>-->
+<!--<link rel='import' href='../bower_components/polymer/lib/elements/dom-repeat.html'>-->
+<dom-module id='geo-location'>
     <script>
-      var loc = document.querySelector('geo-location');
```

Comment:
Want to move this stuff to the README so it gets picked up by webcomponents.org? You can follow https://www.webcomponents.org/publish.